CHANGELOG

```diff
@@ -46,6 +46,7 @@
 * Add Kosovo to the list of countries [AnotherJoSmith] #3226
 * Realex: Adds 3DS 1&2 support through external MPI [filipebarcos] #3284
 * PayPal: Adds 3DS 1 support through external MPI [nebdil] #3279
+* Realex: Re-implement credit as general credit [leila-alderman] #3280
```

Comment:
Move your changelog entry up to HEAD, since this was not included in the July 26 version. (Easy to miss)

Reply:
Thanks for catching that! So grateful for your re-review!